README.md

```diff
@@ -1,7 +1,5 @@
 # vis-charts
 
-**This is a "plug'n & play replacement" for [vis](https://github.com/almende/vis), wich is no longer maintained**
-
```

Comment:
@Thomaash Can you please also remove/update the "Usage" section in this README (and maybe even remove the greenkeeper badge). Thx!